index.html

```diff
@@ -2489,6 +2489,16 @@ <h3 property="schema:name">Social Web Working Group</h3>
         </p>
       </section>
 
+      <section id="security-recursive-objects">
+        <h2>Recursive Objects</h2>
+        <p>
+	  Servers should take care to properly handle recursive ActivityStreams
```

Comment:
I think it might be a good idea to open up "properly". What should the implementer do to protect themselves? Or a short example of what could happen if not. At least I'm not sure what I should do to fulfil this paragraph.